ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -26,31 +25,42 @@ import UIKit
 
     @objc
     public func createPaywallView() -> PaywallViewController {
-        let controller = PaywallViewController()
+        let controller = PaywallViewController(dismissRequestedHandler: { controller in
+            self.delegate?.paywallViewControllerRequestedDismissal?(controller)
```

Comment:
As mentioned, I think we might want to weakify `self` here?

Reply:
I think so yes